src/components/expansion/ExpansionRuns.svelte

```diff
@@ -173,6 +174,7 @@
   <CssGridGutter track={1} type="column" />
 
   <SequenceEditor
+    parcel={$parcel}
```

Comment:
Is there a selected parcel on this page?

Reply:
We'll have to select it inside the component when an expansion run is selected.

Reply:
Right, but having `SequenceEditor` pull the `$parcel` store gives no indication of how that store is set. It's just pulling it out of thin air right now. Moving that logic into `ExpansionRuns`, where there already is logic for handling what is selected would be easier to grok as to what drives the derivation of `$parcel`. I think that `SequenceEditor` just needs to know how handle a parcel that it's given.